simplecpp.cpp

```diff
@@ -58,6 +58,15 @@ static bool isOct(const std::string &s)
     return s.size()>1 && (s[0]=='0') && (s[1] >= '0') && (s[1] < '8');
 }
 
+static bool isStringConstant(const std::string &s)
+{
+    return s.size() > 1 && (s[0]=='\"') && (*s.rbegin()=='\"');
```

Comment:
I think this works also: `s.back() == '\"'`

Reply:
Yeah, that's what I'd prefer but apparently `back()` is c++11 and the compiler complained because the use c++03.

Reply:
And by "the" I mean "the test makefile target". So while that's a bit ugly, using `-std=c++03` does indeed give `error: no member named 'back' in 'std::basic_string…` (although admittedly not with every compiler I tried).

Reply:
ok!